ably/readme_examples_test.go

```diff
@@ -1,13 +1,18 @@
 // Generated by test_readme_examples. DO NOT EDIT
 package ably_test
 
-import "testing"
-import "github.com/ably/ably-go/ably"
-import "github.com/ably/ably-go/ably/ablytest"
+import (
```

Comment:
Please run `go generate ./...` and leave this leave unaltered afterwards. CI is failing because of this.